components/brave_wallet_ui/options/wyre-asset-options.ts

```diff
@@ -0,0 +1,155 @@
+import { AssetOptionType } from '../constants/types'
+import {
+  ETHIconUrl,
+  BATIconUrl
+} from '../assets/asset-icons'
+import {
+  AaveIconURL,
+  BusdIconURL,
+  CompIconURL,
+  CrvIconURL,
+  DaiIconURL,
+  GusdIconURL,
+  LinkIconURL,
+  MkrIconURL,
+  MusdcIconURL,
+  PaxIconURL,
+  PdaiIconURL,
+  SnxIconURL,
+  UmaIconURL,
+  UniIconURL,
+  UsdcIconURL,
+  UsdsIconURL,
+  UsdtIconURL,
+  WbtcIconURL,
+  YfiIconURL
+} from '../assets/wyre-asset-icons'
+
+export const WyreAssetOptions: AssetOptionType[] = [
+  {
+    id: '1',
```

Comment:
Why do we need to hardcode the `id`s?

Reply:
`AssetOptionType[]` was set up to have an `id` for each assets `contractID`. Since this list of assets needs to be displayed in places that is expecting type of `AssetOptionType[]` i needed to give it some sort of `id`. This will also be refactored when the `PR` for this issue lands. https://github.com/brave/brave-browser/issues/17080